lib/metric_fu/metrics/reek/generator.rb

```diff
@@ -1,6 +1,6 @@
 module MetricFu
   class ReekGenerator < Generator
-    REEK_REGEX = /^(\S+) (.*) \((.*)\)$/
+    REEK_REGEX = /^\s*\[([^:]+)\]:(\S+)\s+(.*)\s+\((.*)\)$/
```

Comment:
I assume when you say 'read' you mean 'mentally parse'... the comments let you read it :)